strapi/strapi
# [CTB] Migrate CTB to typescript

### What does it do?

Converts the content-type-builder to typescript

### Why is it needed?

Part of the internal Strapi typescript migration

### How to test it?

- CTB should all work as before
- yarn setup should build
- tests should pass
- there should not be any breaking changes in the publicly exposed types
- strapi typescript projects should still build without any breaking changes in the types

### Related issue(s)/PR(s)
